contracts/tokens/Xcert.sol

```diff
@@ -52,14 +54,11 @@ contract Xcert is Ownable, ERC721, ERC721Metadata, ERC165 {
 
   /*
    * @dev Mapping from NFToken ID to proof.
+   * @notice token Proof has to exist for every token.
```

Comment:
This is a tautology. Because of EVM, proof is automatically initialized to a zero-length list. Perhaps this was meant to say:

> The Proof array for every token must include one or more items